stdlib/doc/sphinx/index.latex.rst

```diff
@@ -0,0 +1,27 @@
+==================================
+ The Rocq Prover Reference Manual
```

Comment:
It looks like the general title of the Stdlib refman in LaTeX was not changed to something more adequate.